common-js/src/main/kotlin/model/helpers.kt

```diff
@@ -0,0 +1,17 @@
+package js.model 
+
+import kotlinx.serialization.*
```

Comment:
This line doesn't look necessary (?) And if it is, you should remove the following line instead